Client.xcodeproj/project.pbxproj

```diff
@@ -5807,7 +5807,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "@executable_path/../../Frameworks";
 				OTHER_SWIFT_FLAGS = "-DMOZ_CHANNEL_BETA -DMOZ_TARGET_TODAY";
 				PRODUCT_BUNDLE_IDENTIFIER = "$(MOZ_BUNDLE_ID).$(PRODUCT_NAME:rfc1034identifier)";
-				PRODUCT_NAME = Today;
+				PRODUCT_NAME = "Firefox - Search";
```

Comment:
Any particular reason we had to change the name here?

Reply:
@nbhasin2 In the UX feedback from Bram, We decided to have the widget named as "Firefox - Search" instead of "Firefox" (the app name) so I changed it in the info.plist of the widget extension. should I change in a different way?

Reply:
Gotcha, think its fine to keep the new name. Although I'll need to double check if the product name is localized 

$(MOZ_BUNDLE_DISPLAY_NAME)